analysis/aws_rules_s3_access_logs/aws_s3_access_error.py

```diff
@@ -1,2 +1,2 @@
 def rule(event):
-    return event['errorCode'] != '-'
+    return 'errorCode' in event
```

Comment:
why not just add the `.get()` call here? We still want to make sure the error is not `-`

Reply:
@rleighton said they normalized the S3 access logs, and converted `-` values to empty values and so those fields get dropped. I guess we could do:

`return event.get('errorCode') != ''`

Reply:
Ah, I missed that. Nevermind then!